packages/modular-scripts/package.json

```diff
@@ -9,6 +9,7 @@
     "node": ">=12"
   },
   "scripts": {
+    "modular": "ts-node src/cli.ts",
```

Comment:
why?

Reply:
This is just useful if you're hanging out in the `modular-scripts` folder itself rather than needing two prompts for each place in the repo.